src/quo2/components/selectors/selectors/style.cljs

```diff
@@ -24,7 +24,7 @@
 (defn checkbox-background-color
   [customization-color]
   {:normal {:checked   (colors/custom-color-by-theme customization-color 50 60)
-            :unchecked (colors/theme-colors colors/white-opa-40 colors/neutral-80-opa-40)}
+            :unchecked (colors/theme-colors colors/white-opa-30 colors/neutral-80-opa-40)}
```

Comment:
Hi @jo-mut, Thank you very much for PR.
I am not sure if am looking into right file, but I don't see white-opa-30 in [selectors figma compoent](https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=398-18229&mode=design&t=X8QmqSRi4YZXqkRB-0)
But I do see white-opa-40 for disabled, unchecked variant.

<img src="https://github.com/status-im/status-mobile/assets/17097240/47540a2e-b00e-49b1-85be-ce4bad383fa6" height=200/>

if these changes refer to some other file, please link it

Reply:
@Parveshdhull actually I will have to revert this change. The figma design i was referencing has been updating and this change is no longer valid. Its possible that I missed the correct color value. Thanks for confirming